packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import throttle from 'lodash/throttle';
 import styled, { keyframes, css } from 'styled-components';
 import type { FlattenSimpleInterpolation } from 'styled-components';
-import { autoUpdate, useFloating } from '@floating-ui/react';
+import { autoUpdate, detectOverflow, useFloating } from '@floating-ui/react';
```

Comment:
Maybe we should update our eslint rule for this case

Reply:
How? 
1. Its okay to import type from a web package in a common file
2. Its okay to import anything platform specific in a platform-specific file

Wont this be a complex check?